composer.json

```diff
@@ -0,0 +1,82 @@
+{
+    "name": "oomphinc/drupal-scaffold",
+    "description": "A Drupal 8 scaffold project",
+    "type": "project",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nathan Dentzau",
+            "email": "nathan.dentzau@gmail.com"
+        }
+    ],
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        },
+        {
+            "type": "composer",
+            "url": "https://asset-packagist.org"
+        }
+    ],
+    "require": {
+        "bower-asset/bootstrap": "^4.0",
+        "composer/installers": "^1.5",
+        "cweagans/composer-patches": "^1.6",
+        "drupal-composer/drupal-scaffold": "^2.4",
+        "drupal-composer/preserve-paths": "^0.1.4",
+        "drupal/admin_toolbar": "^1.23",
+        "drupal/coder": "^8.2",
+        "drupal/coffee": "^1.0",
+        "drupal/config_installer": "^1.8",
+        "drupal/config_split": "^1.3",
+        "drupal/core": "^8.5",
+        "drupal/honeypot": "^1.27",
+        "drupal/oomph_paragraphs": "^1.0",
+        "drupal/recreate_block_content": "^2.0",
```

Comment:
Do we want to use https://www.drupal.org/project/fixed_block_content instead?

Reply:
Might be too early to say we're switching to that. I've only exported 2 blocks with that module so far.

Reply:
I would say if it carries over content it's better than recreate block content and we should consider adding it.